gen/modules/general.h

```diff
@@ -279,3 +281,33 @@ struct user_desc {
 #define BLKSSZGET  0x1268
 #define BLKPBSZGET 0x127B
 #endif
+
+
+/* -- based on  sys/socket.h -- */
```

Comment:
"sys/socket.h" is a libc header file name, and this file is describing the kernel ABI. These specific types may be the same between the two, but overall the distinction between kernel ABI and libc ABI is subtle enough that it's worth being careful about. So I suggest just removing this comment.

Reply:
done